p2p/simulations/network.go

```diff
@@ -665,6 +680,12 @@ func (self *Network) Load(snap *Snapshot) error {
 		}
 	}
 	for _, conn := range snap.Conns {
+
+		if !self.GetNode(conn.One).Up || !self.GetNode(conn.Other).Up {
+			//in this case, at least one of the nodes of a connection is not up,
+			//so it would result in the snapshot `Load` to fail
+			continue
+		}
```

Comment:
this really should not happen, I got the feeling it should be corrected in the snapshot code

Reply:
OK - I vote to keep this code nonetheless, as it is a consistency check worth keeping